splinterd/src/daemon.rs

```diff
@@ -246,10 +247,17 @@ impl SplinterDaemon {
         let admin_service_verifier = secp256k1_context.new_verifier();
         let auth_config_verifier = secp256k1_context.new_verifier();
         let signing_context = Arc::new(Mutex::new(secp256k1_context));
+        let node_id: String = crate::node_id::get_node_id(
```

Comment:
Add a use statement above for get_node_id instead of useing the whole crate::node_id::get_node_id